arch/arm/core/aarch32/cortex_m/Kconfig

```
@@ -96,6 +96,13 @@ config DCACHE_LINE_SIZE
 config ICACHE_LINE_SIZE
 	default 32
 
+config DEBUG_MONITOR
+	bool "Use debug monitor interrupt for debugging"
+	depends on !CPU_CORTEX_M0
```

This is too specific. This needs to be:

```suggestion
	depends on !ARMV6_M_ARMV8_M_BASELINE
```